news/2019-03-10-project-loved-week-of-march-10th.md

```diff
@@ -0,0 +1,207 @@
+
+[![](/wiki/shared/news/2019-03-10-project-loved-week-of-march-10th/osu/60th-summer-of-love.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)
+
+[forest - 60th Summer of Love](https://osu.ppy.sh/beatmapsets/803415#osu) by [Fushimi Rio](https://osu.ppy.sh/users/1205412)  
+*written by [DigitalHypno](https://osu.ppy.sh/users/4384207)*
+
+Many of you may already recognize the mapset owner, Fushimi Rio, for their various ranked maps over the last few years. Their mapset of "forest - 60th Summer of Love" features three difficulties: a 3-star, a 5-star, and a 7-star, the lower two being guest difficulties. With a BPM of 158, this catchy song lends itself to a very alternate-heavy mapping style, which can be seen in the highest difficulty. The two lower difficulties feature more conventional mapping, catering the mapset to all kinds of players. Whether you want to challenge your mechanics or just improve your general gameplay skill, this mapset has a difficulty for you to try—give it a go!
+
+[![](/wiki/shared/news/2019-03-10-project-loved-week-of-march-10th/osu/dash-hopes-3.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)
+
+[Renard - Dash Hopes 3](https://osu.ppy.sh/beatmapsets/53059#osu) by [Rizumu Tenshi](https://osu.ppy.sh/users/865703)  
+*written by [-Roxas](https://osu.ppy.sh/users/1986262)*
+
+Now here's a big boy map appearing in voting, Rizumu Tenshi (formerly Baka Tenshi)'s "Renard - Dash Hopes 3". It's far from the easier side of maps and more close to the "completely insane" side. This map is not for the faint of heart, as it is an intense workout for the fingers: if you didn't know already, this map is just 300 BPM burst streams, and if that wasn't scary enough, the star rating ranges from 8.3 to 9.5. If you do give this map a bash, be sure you are very prepared before taking on this monster!
+
+[![](/wiki/shared/news/2019-03-10-project-loved-week-of-march-10th/osu/planet-shaper.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)
+
+[Camellia - PLANET//SHAPER](https://osu.ppy.sh/beatmapsets/611729#osu) by [09kami](https://osu.ppy.sh/users/443031) and [Leader](https://osu.ppy.sh/users/631530)  
+*written by [-Roxas](https://osu.ppy.sh/users/1986262)*
+
+We're back with another fun 09kami map, this time being "Camellia - PLANET//SHAPER". This map, similar to his other maps, is very slider and tech heavy. In addition, this map also has 2B patterns chucked in throughout the map, making it a bit more unique than the rest. If this is your type of map, then what are you waiting for? Give it a try! With the map being favorites by many players, including notable names like [rrtyui](https://osu.ppy.sh/users/352328), [Akali](https://osu.ppy.sh/users/2127359) and [yf\_bmp](https://osu.ppy.sh/users/1243669), you won't be let down. The star rating does hit 7, so it may be a bit of a challenge, but don't be afraid! You never know how well you'll do until you play it.
```

Comment:
`does hit` -> `almost hits`
```suggestion
We're back with another fun 09kami map, this time being "Camellia - PLANET//SHAPER". This map, similar to his other maps, is very slider and tech heavy. In addition, this map also has 2B patterns chucked in throughout the map, making it a bit more unique than the rest. If this is your type of map, then what are you waiting for? Give it a try! With the map being favorites by many players, including notable names like [rrtyui](https://osu.ppy.sh/users/352328), [Akali](https://osu.ppy.sh/users/2127359) and [yf\_bmp](https://osu.ppy.sh/users/1243669), you won't be let down. The star rating almost hits 7, so it may be a bit of a challenge, but don't be afraid! You never know how well you'll do until you play it.
```